apps/condo/domains/common/components/Pickers/DatePicker.tsx

```diff
@@ -1,7 +1,53 @@
+import React from 'react'
 import { Dayjs } from 'dayjs'
 import dayjsGenerateConfig from 'rc-picker/lib/generate/dayjs'
-import generatePicker from 'antd/lib/date-picker/generatePicker'
+import generatePicker, {
+    PickerDateProps,
+    PickerProps,
+    PickerTimeProps,
+    RangePickerProps,
+} from 'antd/lib/date-picker/generatePicker'
+import { useIntl } from 'react-intl'
+import ruRU from 'antd/lib/locale/ru_RU'
+import enUS from 'antd/lib/locale/en_US'
```

Comment:
you need to move it to `index`

Reply:
It looks like we already have such code

Reply:
Please avoid to create a new places where you need to add a new language

Reply:
fixed it, locale get from configProvider